### ISSUE TITLE
Cant play Video from assets

### ISSUE THREAD

**Issue by MarcelHH** [jhomlala/betterplayer#463]
Hey i use this code 

```
 AspectRatio(
                aspectRatio: 16 / 9,
                child: BetterPlayer.file(
                  "assets/video.mp4",
                  betterPlayerConfiguration: BetterPlayerConfiguration(
                    aspectRatio: 16 / 9,
                  ),
                ),
              ),
```

But Video cant be played :(
`
[ERROR:flutter/shell/common/shell.cc(242)] Dart Unhandled Exception: PlatformException(VideoError, Video player had error com.google.android.exoplayer2.ExoPlaybackException: Source error, null, null), stack trace:`

**Comment by jhomlala** [comment]
@MarcelHH Please use issue template next time. You're trying to play video from assets folder with file source type. Better Player doesn't offer at this moment playing videos from asset. You should save your video in your app directory and then play it from that file. Check example app (welcome_page.dart) especially to see how to save asset to file.

**Comment by VladimirCores** [comment]
Strange, the issue is closed, but with version `better_player: ^0.0.83` I still have that error
```
E/flutter (25143): [ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: PlatformException(VideoError, Video player had error com.google.android.exoplayer2.ExoPlaybackException: Source error, , null)
```
